tools/mo/openvino/tools/mo/front/tf/loader.py

```diff
@@ -226,6 +226,14 @@ def tf_function(x):
 
         conc_func = tf_function.get_concrete_function(model_inputs)
         return freeze_tf2_concrete_function(model, conc_func, env_setup)
+    if isinstance(model, tf.types.experimental.GenericFunction):
```

Comment:
It looks that `tf.types.experimental.GenericFunction` is introduced recently in `2.4`. Can we leave MO compatible with early TF versions please?

Reply:
I double checked it, `GenericFunction `and `ConcreteFunction `was introduced in 2.6. I added checks of TF version for these types.

Reply:
this is experimental type. May be it makes sense to check stable API. Is it possible? Please check